operate/qa/util/src/main/java/io/camunda/operate/qa/util/TestContainerUtil.java

```diff
@@ -524,7 +525,8 @@ public ZeebeContainer startZeebe(final String version, final TestContext testCon
           .withEnv("ZEEBE_CLOCK_CONTROLLED", "true")
           .withEnv("ZEEBE_BROKER_DATA_DISKUSAGEREPLICATIONWATERMARK", "0.99")
           .withEnv("ZEEBE_BROKER_DATA_DISKUSAGECOMMANDWATERMARK", "0.98")
-          .withEnv("ZEEBE_BROKER_DATA_SNAPSHOTPERIOD", "1m");
+          .withEnv("ZEEBE_BROKER_DATA_SNAPSHOTPERIOD", "1m")
+          .withEnv(AuthenticationProperties.getAllowUnauthenticatedApiAccessEnvVar(), "true");
```

Comment:
🔧 given it's the default for now, I think we can remove that here + the dependency on the auth module in the pom.
```suggestion
          .withEnv("ZEEBE_BROKER_DATA_SNAPSHOTPERIOD", "1m");
```